progress/meter.go

```diff
@@ -58,15 +105,6 @@ func (p *ProgressMeter) Start() {
 	}
 }
 
-// Add tells the progress meter that a transferring file is being added to the
-// TransferQueue.
-func (p *ProgressMeter) Add(name string) {
-	idx := atomic.AddInt64(&p.transferringFiles, 1)
-	p.fileIndexMutex.Lock()
-	p.fileIndex[name] = idx
-	p.fileIndexMutex.Unlock()
-}
-
 // Skip tells the progress meter that a file of size `size` is being skipped
 // because the transfer is unnecessary.
 func (p *ProgressMeter) Skip(size int64) {
```

Comment:
Per our 🍖 -space discussion, I definitely agree that it'd be nice for this function to have a name that agrees with the `AddEstimate` function.

What do you think about `DecrementEstimate` or `DecreaseEstimate`?